src/writers.py

```diff
@@ -80,19 +80,12 @@ def __init__(self, cfg):
         except xmlrpc.client.Error:
             raise UnavailableError('LAVA device is offline')
 
-    def __get_device_status(self, device):
+    def query_device_status(self, device):
         board = "%s_01" % device
 
         return self._con.scheduler.get_device_status(board)
 
```

Comment:
You're adding a new public function that is not being tested.

Moreover, that function is not implemented by all the classes subclassing the Writer class, meaning that if you happen to use another class there (such as FileWriter, used when you have the --no-send option), ctt will crash.

I guess a more fundamental question would be why is that change needed in the first place.